nativelib/src/main/resources/scala-native/stackOverflowGuards.c

```diff
@@ -78,25 +80,27 @@ static struct sigaction *resolvePreviousSignalHandler(int sig) {
         return &previousSignalHandlers[1];
     default:
         fprintf(stderr,
-                "ScalaNative :: StackOverflowHandler does not define handler "
+                "%s StackOverflowHandler does not define handler "
                 "for %d signal\n",
-                sig);
+                snErrorPrefix, sig);
         exit(sig);
     }
 }
 
 static void protectStackGuardPage() {
     if (mprotect(currentThreadInfo.stackGuardPage, resolvePageSize(),
                  PROT_NONE) == -1) {
-        perror("ScalaNative Fatal Error :: StackOverflowHandler guard "
+        perror("ScalaNative Fatal Error: "
+               "StackOverflowHandler guard "
                "protection failed");
         exit(EXIT_FAILURE);
     }
 }
 static void unprotectStackGuardPage() {
     if (mprotect(currentThreadInfo.stackGuardPage, resolvePageSize(),
                  PROT_READ | PROT_WRITE) == -1) {
-        perror("ScalaNative Fatal Error :: StackOverflowHandler guard "
+        perror("ScalaNative Fatal Error: "
```

Comment:
Again here?

Reply:
Fixed in https://github.com/scala-native/scala-native/pull/4310/commits/15cfff33b7cc030553c3c3eddbe4fad62d87de87.